### ISSUE TITLE
[BUG] jest-dev-server won't be installed by default

### ISSUE THREAD

**Issue by thernstig** [mmarkelov/jest-playwright#84]
Hi!

I just tried to install version 0.0.15 and it installs no `requires`, see here:

```json
    "jest-playwright-preset": {
      "version": "0.0.15",
      "resolved": "https://registry.npmjs.org/jest-playwright-preset/-/jest-playwright-preset-0.0.15.tgz",
      "integrity": "sha512-DHWu0fNLzmfudVZ+wOKuMTiCpavu+TicMateKNgajH1A5RzDY78PceAGFrlEQIrnDyTsmhgbruMALgxEhprKcQ==",
      "dev": true
    },
```

This means e.g. `jest-dev-server` is not installed nor no other packages. Am I missing something?

**Comment by mxschmitt** [comment]
I think we should add `jest-dev-server` to the normal dependencies so it will be installed by default.

cc @mmarkelov

**Comment by mmarkelov** [comment]
@mxschmitt may be it should be also peer dep?

**Comment by mxschmitt** [comment]
> @mxschmitt may be it should be also peer dep?

Either that (but then it won't be automatically installed right?) or a normal dependency. I would go with the second approach. Of course in the end more to install, but that should be ok (https://unpkg.com/jest-dev-server@4.4.0/package.json)

**Comment by mmarkelov** [comment]
Closed with #89
